docs/phdata.rst

```diff
@@ -187,14 +189,14 @@ period) are obtained by applying one of these two conditions:
 Detectors specs
 """""""""""""""
 
-Inside **measurement_specs**, the sub-group **detectors_specs/**
-contains the mapping between the each pixel ID and the detection channels
-(i.e. spectral bands, polarizations or beam-split channels).
+Within **measurement_specs**, the **detectors_specs/** sub-group 
+contains all (detector ID, detection channel) associations 
+(i.e. spectral bands, polarizations or beam-split channels (**what's that?**)).
 
-Note that a detector ID can be a single integer of a n-tuple of integers,
-to support the case of 2-D detector arrays. Therefore an array of detector
-IDs can be either a 1-D or a 2-D array, in the latter case it is one row
-per detector.
+Note that a detector ID can be a single integer (single spot), or a n-tuple of integers
+in the case of 2-D detector arrays (multiple spots). Consequently, the detector
+IDs array can be either a 1-D or a 2-D array. In the latter case, it is one row
+per detector (**spot?**).
 
```

Comment:
Even in multi-spot a detector ID can be a single integer. When we number the detectors from 0 to 15, regardless of their geometrical configuration, this is the case.

Will correct